app-creativity/inochi2d-session/autobuild/defines

```diff
@@ -0,0 +1,11 @@
+PKGNAME=inochi2d-session
+PKGSEC=graphics
+PKGDES="Application that allows streaming with Inochi2D puppets"
+PKGDEP="sdl2 mesa lua freetype gcc-runtime dbus desktop-file-utils"
```

Comment:
```
$ inochi-creator 
inochi-creator: error while loading shared libraries: libphobos2-ldc-shared.so.108: cannot open shared object file: No such file or directory
```

Reply:
Seems that we need to rebuild this against LDC 1.38. @jiegec

Reply:
Missing liblphobos dependency, done